hastexo/utils.py

```diff
@@ -21,6 +24,8 @@
     "suspend_timeout": 120,
     "suspend_interval": 60,
     "suspend_concurrency": 4,
+    "delete_age": 14,
```

Comment:
I'm never a big fan of mixing time units, because that sort of thing tends to get very confusing. I suggest this should be defined in seconds as well (even though that means we'll get pretty big numbers).